.github/workflows/buildtest.yml

```diff
@@ -56,3 +56,42 @@ jobs:
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           file: test/coverage/lcov.info
+
+  sriov-operator-e2e-test:
+    name: SR-IOV operator e2e tests
+    needs: [ build-test ]
+    runs-on: [ sriov ]
+    env:
+      TEST_REPORT_PATH: k8s-artifacts
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+
+      - name: build sriov-cni image
+        run: podman build -t ghaction-sriov-cni:pr-${{github.event.pull_request.number}} .
+  
+      - name: Check out sriov operator's code
+        uses: actions/checkout@v2
+        with:
+          repository: zeeke/sriov-network-operator-1
+          ref: e2e-inject-images
+          path: sriov-network-operator-wc
+
+      - name: fix sriov-device-plugin
```

Comment:
why do we need this? the image used by the operator for the device plugin is not good?

Reply:
We don't need this. I'll remove it as soon as the operator's PR is merged:
- https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/518